dotnet/src/Connectors/Connectors.AI.OpenAI/AzureSdk/ChatResult.cs

```diff
@@ -23,8 +23,8 @@ public ChatResult(ChatCompletions resultData, ChatChoice choice)
 
     public ModelResult ModelResult { get; }
 
-    public Task<ChatMessageBase> GetChatMessageAsync(CancellationToken cancellationToken = default)
-        => Task.FromResult<ChatMessageBase>(new SKChatMessage(this._choice.Message));
+    public Task<SemanticKernel.AI.ChatCompletion.ChatMessage> GetChatMessageAsync(CancellationToken cancellationToken = default)
```

Comment:
+1 to the reasons why moving the Base abstraction class to non abstract (clashing the names within other packages)